webapp/src/routes/Signup/Signup.js

```diff
@@ -102,6 +102,7 @@ const Signup = () => {
 
   const handleSetField = useCallback((field, value) => {
     setUser({ [field]: value })
+    console.log(value)
```

Comment:
please remove the `console.log` it's a bad practice commit logs that were added just for debugging propourse